SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
+     * @param   fileOut <String> storing the file name of the output (SFSP)
+     * @param   strategy <String> storing the method of sampling the random dataset
+     * @param   datasetFilePaths <String[]> storing the file paths of the dataset files
+     * @param   numSfsp <Integer> storing the number of statistically significant sequential patterns
+     * @param   numFsp  <Integer> storing the number of frequent sequential patterns
```

Comment:
Do we need these two variables? Can't we just use the `size` of the two data structures below (honest question, I don't know if it is just much more convenient to have the sizes cached in these variables) ?

Reply:
Not sure. Let's discuss when we meet today!